### PR TITLE
Added some tips

### PR DESCRIPTION
**Pull request by RannyBergamotte**
Added some audio trace and other tips.

### CODE REVIEW

File: src/plugins/show-tips/tips.ts [hunk 1]
```diff
@@ -160,13 +160,50 @@ const tips: TipData[] = [
     learnMore:
       "https://help.desmos.com/hc/en-us/articles/4405901719309-Saving-and-Managing-Graphs",
   },
+  // From flexabrotnt https://discord.com/channels/655972529030037504/663607163532476426/999501013096616036
+  {
+    desc: "Extract the x or y coordinate of points by appending .x or .y to your point variable",
+    learnMore:
+      "link",
```
Comment (jared-hughes) [inline src/plugins/show-tips/tips.ts:167]: Is there a relevant hyperlink? otherwise no learnMore field

File: src/plugins/show-tips/tips.ts [hunk 2]
```diff
@@ -160,13 +160,50 @@ const tips: TipData[] = [
...
+    learnMore:
+      "https://www.desmos.com",
+  },
+  {
+    desc: "Don't want your viewport to be moved? Lock it in the graph settings!",
+    learnMore:
+      "https://help.desmos.com/hc/en-us/articles/4405296853517-Graph-Settings",
+  },
+  {
+    desc: "Enable GLEsmos to make implicits run faster",
```
Comment (jared-hughes) [inline src/plugins/show-tips/tips.ts:192]:
```suggestion
    desc: "Enable the GLesmos plugin to make some implicits run faster",
```

File: src/plugins/show-tips/tips.ts [hunk 3]
```diff
@@ -160,13 +160,50 @@ const tips: TipData[] = [
     learnMore:
       "https://help.desmos.com/hc/en-us/articles/4405901719309-Saving-and-Managing-Graphs",
   },
+  // From flexabrotnt https://discord.com/channels/655972529030037504/663607163532476426/999501013096616036
```
Comment (jared-hughes) [inline src/plugins/show-tips/tips.ts:163]:
```suggestion
  // From flexabrotnt
```

"From flexabrotnt" is enough, discord link isn't needed

**Review by jared-hughes — CHANGES_REQUESTED** [review]
Great tips! I've put some suggestions for changes.

Also, to deal with the format error, run `npm run prettier` assuming you have npm installed, or just copy into the left side of https://prettier.io/playground and paste the formatted version from the right side

File: src/plugins/show-tips/tips.ts (hunk 1 above)
Comment (RannyBergamotte) [reply src/plugins/show-tips/tips.ts:167]: My bad.
I tried looking for one on the help desmos page, but didn't find any.

**Review by jared-hughes — APPROVED** [review]
Looks good! I went ahead and applied Prettier directly.

**Pull request merged**